test/integration/integration_test.go

```diff
@@ -1139,3 +1170,14 @@ func setupSignalHandler() (context.Context, context.CancelFunc) {
 
 	return ctx, cancel
 }
+
+// getEnvInt reads an integer value from an environment variable.
+// If the env variable is not set or if it's not a valid integer, it returns the default value.
+func getEnvInt(key string, defaultValue int) int {
```

Comment:
Does this function handle negative integers? What should we do in that case?

Reply:
It does. If it's set to a negative vault, it will return as is. If the environment variable is not set or if the converstion fails, it will return the `defaultValue`.